config/routes.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  resources :strategies
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-  root 'pages#home'
+  root 'strategies#index'
```

Comment:
Routes should be ordered in popularity order. 'root' on the top, it's the most used route in the App.